src/http/kv/mod.rs

```diff
@@ -0,0 +1,93 @@
+mod create_namespace;
+
+pub use create_namespace::create_namespace;
+
+use cloudflare::auth::Credentials;
+use cloudflare::response::{APIError, APISuccess};
+use cloudflare::workerskv;
+use cloudflare::HTTPAPIClient;
+
+use crate::settings::global_user::GlobalUser;
+
+impl From<GlobalUser> for Credentials {
+    fn from(user: GlobalUser) -> Credentials {
+        Credentials::User {
+            key: user.api_key,
+            email: user.email,
+        }
+    }
+}
+
+fn new_api_client(user: GlobalUser) -> HTTPAPIClient {
```

Comment:
nit incoming: We can extend the HTTPAPIClient type and implement From<GlobalUser> for it. `new_api_client` takes ownership of `user` so the semantics are the same.

I would suggest the pattern of `use cloudflare::HTTPAPIClient as CFAPIClient` if we did this, and then `CFAPIClient::from(user)` which _feels_ kinda nice

Reply:
Not sure how I feel about this. `From`'s semantics imply conversion. Although admittedly isomorphism is not a part of the definition, it doesn't feel right to say that we're _converting_ a user into an API client. A constructor to me feels more appropriate here.